src/modules/clearComplete.js

```diff
@@ -0,0 +1,12 @@
+import getTodo from './getTodo.js';
+
+const clearComplete = () => {
+  const clearButton = document.getElementById('clear-complete');
+  clearButton.addEventListener('click', () => {
+    const todos = getTodo().todo;
+    const newlist = todos.filter((item) => item.completed !== true);
+    localStorage.setItem('todo', JSON.stringify(newlist));
+    window.location.reload();
+  });
+};
```

Comment:
- [OPTIONAL] It would be nice if the task indexes were updated after user interaction. Sometimes, after clearing all completed and adding a new task, the app can have tasks with the same index and it can cause a bad user experience.
 -
![Untitled (1)](https://user-images.githubusercontent.com/61919617/184388149-c287ec83-7866-4eeb-8038-e934b00b9571.gif)

Reply:
Thank you @leonardodiasb that's a good ide, I'll make sure to implement it.